### PR TITLE
Features add controller with tests for Service

### PR DESCRIPTION
**Pull request by illiakozr**
add controller with tests for Service

### CODE REVIEW

File: VetClinic.API.Tests/Controllers/ServiceControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        private void InitializeMapper()
+        {
+            var profile = new ServiceProfile();
+            var configuration = new MapperConfiguration(cfg => cfg.AddProfile(profile));
+            _mapper = new Mapper(configuration);
+        }
+
+        private List<Service> GetTestServices()
+        {
+            return new List<Service>(){
```
Comment (Serafinium) [inline VetClinic.API.Tests/Controllers/ServiceControllerTest.cs:39]: Please use autofixture in future

File: VetClinic.API/Controllers/ServiceController.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+using System.Threading.Tasks;
+using VetClinic.API.DTO;
+using VetClinic.API.DTO.Service;
+using VetClinic.BLL.Services.Interfaces;
+using VetClinic.DAL.Entities;
+
+namespace VetClinic.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```
Comment (Serafinium) [inline VetClinic.API/Controllers/ServiceController.cs:13]: Please add postfix Async for all controllers

File: VetClinic.API/Controllers/ServiceController.cs [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        public ServiceController(IMapper mapper, IServiceService serviceService)
+        {
+            _mapper = mapper;
+            _serviceService = serviceService;
+        }
+
+        //GET api/service
+        [HttpGet]
+        public async Task<ActionResult<ICollection<ServiceDto>>> Index()
```
Comment (Serafinium) [inline VetClinic.API/Controllers/ServiceController.cs:27]: Please use CRUD naming

File: VetClinic.BLL.Tests/Services/ServiceServiceTest.cs [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            // Arrange
+            var services = GetTestServices();
+            _wrapper.Setup(s => s.ServiceRepository.GetAsync(null,null,null,false)).ReturnsAsync(services);
+
+            // Act 
+            var result = await _serviceService.GetAllServicesAsync();
+
+            // Assert
+            Assert.IsAssignableFrom<ICollection<Service>>(result);
+            Assert.Equal(3, result.Count);
```
Comment (Serafinium) [inline VetClinic.BLL.Tests/Services/ServiceServiceTest.cs:45]: Please don't use magic numbers
Use explicit initialization

File: VetClinic.DAL/Configurations/AppointmentsConfiguration.cs [hunk 5]
```diff
@@ -8,7 +8,7 @@ namespace VetClinic.DAL.Configurations
     public class AppointmentsConfiguration : IEntityTypeConfiguration<Appointment>
     {
         public void Configure(EntityTypeBuilder<Appointment> builder)
-        {
+        {
```
Comment (Serafinium) [inline VetClinic.DAL/Configurations/AppointmentsConfiguration.cs:11]: Format the code

File: VetClinic.DAL/Repositories/Interfaces/IRepositoryWrapper.cs [hunk 6]
```diff
@@ -14,6 +14,6 @@ public interface IRepositoryWrapper
         IProcedureRepository ProcedureRepository { get; }
         IServiceRepository ServiceRepository { get; }
         IStatusRepository StatusRepository { get; }
-        Task SaveAsync();
+        Task<int> SaveAsync();
```
Comment (Serafinium) [inline VetClinic.DAL/Repositories/Interfaces/IRepositoryWrapper.cs:17]: Why do you use generic?

**Review by yaromeister — APPROVED** [review]

**Pull request merged**